### ISSUE TITLE
Can pairing the device be a separate - explicit - step?

### ISSUE THREAD

**Issue by zbynekwinkler** [palinek/pvr.sledovanitv.cz#55]
I ran into a problem when sledovanitv returned "pairing limit reached" when trying to use this addon. I had a problem with my local network which frequently failed. IIUC the plugin tries to pair again the current device when login fails (for example due to network problem). This resulted in "tons" of kodi devices registered on the site. Even when I deleted all of them, fixed my local network, still the same error. Finally I had to ask sledovanitv to reset the limit (I've learned it is a monthly limit).

I believe an improved workflow would be to pair the device only once, when the config dialog is filled. This would prevent ever again reaching the pairing limit, even when there are network problems.

**Comment by palinek** [comment]
Pairing workflow was updated similar way in e4fdc739f5249804de1a8c88d2179a1e4fd3b246. Addon will not try to re-pair until reasonable error comes from network. And even it will never create new pairing until previous one successfully deleted.

**Comment by zbynekwinkler** [comment]
We don't have documentation to the API but when deleting all pairings manually using the website didn't make the error go away, that seems to suggest that deleting pairing before creating new one, might have no effect wrt the montly pairing limit.

Since pairing is now limited, it would be nice to have this under control of the user (like "pair this device" button). If anything, we would know sooner that something is broken somewhere when the re-pairing is done often. Having effective ban from the network is somewhat harsh notification. It would be nice to have some warning signs along the way.
